tests/test_secrets_awssm.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2021 Chris Rose
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+
+import attr
+import boto3
+import pytest
+
+from moto import mock_secretsmanager
+
+import environ
+
+from environ.exceptions import MissingSecretError
+from environ.secrets import SecretsManagerSecrets, _SecretStr
+
+
+@pytest.fixture(autouse=True, scope="session")
+def shut_boto_up():
```

Comment:
```suggestion
@pytest.fixture(name="shut_boto_up", autouse=True, scope="session")
def _shut_boto_up():
```

I know this is not consistently throughout the code base, but I'd prefer new fixtures to be written this way.

(it prevents the common mistake of not declaring the fixture in the signature and getting weird errors working with the function)